modules/phase_field/include/action/PolycrystalElDrForceAction.h

```diff
@@ -0,0 +1,30 @@
+#ifndef POLYCRYSTALELDRFORCEACTION_H
+#define POLYCRYSTALELDRFORCEACTION_H
+
+#include "Action.h"
+
+// Forward Declarations
+class PolycrystalElDrForceAction;
```

Comment:
I would vote for changing ```ElDr``` to ```ElasticDriving```. I'm not a fan of abbreviations in the class names. A few extra characters don't cost us anything.